src/components/CohortBuilder/Summary/index.js

```diff
@@ -44,38 +43,45 @@ const Summary = ({
     ]}
   >
     {({ isLoading, data }) => {
-      const [demographicData, ageDiagData, studiesData, topDiagnosesData, fileDataTypes] =
-        data || [];
+      const [
+        demographicData = [],
+        ageDiagData = [],
+        studiesData = [],
+        topDiagnosesData = [],
+        fileDataTypes = [],
+      ] = data;
 
-      return isLoading ? (
-        <Row nogutter>
-          <div className={theme.fillCenter} style={{ marginTop: '30px' }}>
-            <LoadingSpinner color={theme.greyScale11} size={'50px'} />
-          </div>
-        </Row>
-      ) : !data ? (
+      return !data ? (
         <Row nogutter> no data</Row>
       ) : (
         <Row nogutter>
           <Col xl={12}>
             <Row nogutter>
               <PaddedColumn md={spacing.md} lg={spacing.lg}>
-                <FileBreakdown fileDataTypes={fileDataTypes} sqon={sqon} />
+                <FileBreakdown
+                  fileDataTypes={fileDataTypes}
+                  sqon={sqon}
+                  isParentLoading={isLoading}
+                />
               </PaddedColumn>
               <PaddedColumn md={spacing.md} lg={spacing.lg}>
-                <StudiesChart studies={studiesData} sqon={sqon} />
+                <StudiesChart studies={studiesData} sqon={sqon} isParentLoading={isLoading} />
               </PaddedColumn>
               <PaddedColumn md={spacing.md} lg={spacing.lg}>
-                <DiagnosesChart sqon={sqon} topDiagnoses={topDiagnosesData} />
+                <DiagnosesChart
+                  sqon={sqon}
+                  topDiagnoses={topDiagnosesData}
+                  isParentLoading={isLoading}
+                />
               </PaddedColumn>
               <PaddedColumn md={spacing.md} lg={spacing.lg}>
-                <DemographicChart data={demographicData} />
+                <DemographicChart data={demographicData} isParentLoading={isLoading} />
               </PaddedColumn>
               <PaddedColumn md={spacing.md} lg={spacing.lg}>
-                <SurvivalChart data={survivalPlotMock} />
+                <SurvivalChart data={survivalPlotMock} isParentLoading={isLoading} />
               </PaddedColumn>{' '}
               <PaddedColumn md={spacing.md} lg={spacing.lg}>
-                <AgeDiagChart data={ageDiagData} />
+                <AgeDiagChart data={ageDiagData} isParentLoading={isLoading} />
```

Comment:
Does it make more sense to expose `isParentLoading` as just `isLoading` instead? Totally just a naming thing, but I think it reads better :)
Each card can rename it internally to account for namespacing

Reply:
Sure!